### ISSUE TITLE
error while installing 

### ISSUE THREAD

**Issue by azr-arch** [Eyevinn/node-srt#62]
i cant install install the module 
![Screenshot 2023-09-18 115836](https://github.com/Eyevinn/node-srt/assets/79351619/53b3c045-7f59-489a-a1c9-c7af1c5e9309)

**Comment by neoxpert** [comment]
As stated in the build log, it seems that you do not have a working [vcpkg](https://vcpkg.io/en/) installation. vcpkg is utilized by [scripts/build-srt-sdk.js ](https://github.com/Eyevinn/node-srt/blob/23407cea603fb15613c26ea7f0528b861f086312/scripts/build-srt-sdk.js) when installing / retrieving depdendencies like OpenSSL etc. when building for Windows.

**Comment by birme** [comment]
Thank you for reporting this, @azr-arch. A contributor already identified the root cause: you do not have a working vcpkg installation. On Windows, `scripts/build-srt-sdk.js` uses [vcpkg](https://vcpkg.io/en/) to acquire OpenSSL and pthreads before building libSRT, and without it the cmake configure step fails.

To get unblocked on Windows you will need:

1. Install [Visual Studio Build Tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022) with the "Desktop development with C++" workload selected.
2. Install [cmake](https://cmake.org/download/) and ensure it is on your PATH.
3. Install [vcpkg](https://vcpkg.io/en/getting-started) and set the `VCPKG_ROOT` environment variable to its root directory.
4. Re-run `npm install`.

You may also find the related open issues #70 and #72 helpful — they cover the same Windows setup requirements in more detail.

Since this issue is now almost two years old and the fix is well understood, we are closing it to keep the tracker tidy. Please open a new issue if you continue to see problems after following the steps above — include your OS version, Node version, npm version, cmake version, and the full output of `npm install --verbose`. We are happy to help.

Thank you again for trying out `@eyevinn/srt`.

_node-srt maintenance bot — `build-issue`, `platform-windows`; closing stale issue, missing vcpkg on Windows._
